packages/components/src/border-box-control/styles.ts

```diff
@@ -46,14 +52,14 @@ const BorderBoxStyleWithFallback = ( border?: Border ) => {
 
 export const borderBoxControlVisualizer = (
 	borders?: Borders,
-	__next36pxDefaultSize?: boolean
+	__next40pxDefaultSize?: boolean
 ) => {
 	return css`
 		position: absolute;
-		top: ${ __next36pxDefaultSize ? '18px' : '15px' };
-		right: 30px;
-		bottom: ${ __next36pxDefaultSize ? '18px' : '15px' };
-		left: 30px;
+		top: ${ __next40pxDefaultSize ? '20px' : '15px' };
+		right: ${ __next40pxDefaultSize ? '39px' : '30px' };
+		bottom: ${ __next40pxDefaultSize ? '20px' : '15px' };
+		left: ${ __next40pxDefaultSize ? '39px' : '30px' };
```

Comment:
For some reason, the left border is not perfectly aligned on my screen for the old `30px` height.

Applying this change fixes it:

```suggestion
		left: ${ __next40pxDefaultSize ? '39px' : '29px' };
```